includes/wikia/Wikia.php

```diff
@@ -35,10 +35,6 @@
 $wgHooks['SkinTemplateOutputPageBeforeExec'][] = "Wikia::onSkinTemplateOutputPageBeforeExec";
 $wgHooks['UploadVerifyFile']         [] = 'Wikia::onUploadVerifyFile';
 
-# User hooks
-$wgHooks['UserNameLoadFromId']       [] = "Wikia::onUserNameLoadFromId";
-$wgHooks['UserLoadFromDatabase']     [] = "Wikia::onUserLoadFromDatabase";
```

Comment:
I kept the invocation of `UserLoadFromDatabase` and `UserNameLoadFromId` hooks - they come from MediaWiki upstream.